core/dbt/adapters/sql/connections.py

```diff
@@ -15,7 +15,7 @@
 
 
 class SQLConnectionManager(BaseConnectionManager):
-    """The default connection manager with some common SQL methods implemented.
+    """The default connection manager with some common SQL methods was implemented.
```

Comment:
```suggestion
    """The default connection manager with some common SQL methods implemented.
```
Leaving the same since it still is this way